README.md

```diff
@@ -39,6 +39,33 @@ GPU and drivers, so the containerized environments can only focus on user-space
 
 🚧🚧
 
+## Development
```

Comment:
@ca-scribner good point! Let me 
1. convert this then into a sub-section of `Building the snap`, so that it's clear that this is only for this piece
2. add a `Running locally` section, that will be under construction in which we can put in the future the details of how to run the python code locally

WDYT?